compiler/optimizer/Structure.hpp

```diff
@@ -626,8 +626,8 @@ class TR_RegionStructure : public TR_Structure
         void reset() {_iter.SetToFirstOne();}
 
         private:
-          TR::SparseBitVector _nodes;
-          TR::SparseBitVector::Cursor _iter;
+          TR::BitVector _nodes;
+          TR::BitVector::Cursor _iter;
```

Comment:
Ditto to my previous comment; perhaps the `BitVector` `typedef` ought to be in Structure.hpp?